proposed/2022/PrivateAssetIndepdentFromExcludeAsset.md

```diff
@@ -0,0 +1,238 @@
+# Opt-in experience for PrivateAssets and ExcludeAssets options are independent
+
+- Author Name <https://github.com/erdembayar>
+- Start Date 2022-12-07
+- GitHub Issue <https://github.com/NuGet/Home/issues/6938>
+- GitHub PR <https://github.com/NuGet/NuGet.Client/pull/4976>
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+Currently asset consumption via `PrivateAssets` option experience for assets from transitive package in a project/package is not deterministic for parent consuming project, here `PrivateAssets` option calculation is not independent from `IncludeAssets/ExcludeAssets` option, if an asset is not consumed in current project then it doesn't flow up the regardless of `PrivateAssets` option.
```

Comment:
> is not deterministic

I think it is actually deterministic, in that the behavior is defined, consistent and predictable. It just doesn't have the behavior that we sometimes need. In a build, non-deterministic behavior is often a bug, but that isn't really the case here, so I'd suggest changing your wording. Perhaps "is not fully controllable by the project author".